apis/dataprotection/v1alpha1/backup_types.go

```diff
@@ -267,7 +267,10 @@ func GetRecoverableTimeRange(backups []Backup) []BackupLogStatus {
 				continue
 			}
 			startTime = *b.Status.GetStartTime()
-			stopTime = *b.Status.GetStopTime()
+			// the stop time can be nil, because the logfile backup never ends
```

Comment:
better to return a pointer time, because` logfileStartTime.IsZero() || logfileStopTime.IsZero() ` will check it.